### ISSUE TITLE
auto deploy lag profiling on cloud world builds

### ISSUE THREAD

**Issue by j3fffff** [StrangeLoopGames/EcoIssues#19230]
In the build process, cloud world 'release' builds should include the tools for lag profiling:
- dotnet-trace (CPU detail)
- dotnet-counters (General status)
- dotnet-dump (Memory detail)

all staging builds already include this, So I would guess you can copy paste that build process.

'release' steam and S3 builds do not need the lag profiling plugins

**Comment by mirasrael** [comment]
Added this image for 0.9.1.4. strangeloopgames/eco-game-server:0.9.1.4-beta-release-116-dev
I also added script which you can use to make any required version for profiling like:
```
powershell Tools/Docker_MakeDevelopmentImage.ps1 0.9.1.4-beta-release-116
```

**Comment by j3fffff** [comment]
alex did a temporary fix. task is ment for auto deploy

**Comment by WeaselDog** [comment]
This is fixed in staging and will be implemented in 9.1.6
